Sources/KumoCoding/KumoCoding.h

```diff
@@ -0,0 +1,19 @@
+//
```

Comment:
Please delete the header entirely

Reply:
Apologies! I meant the file header comments, I believe the header file itself is necessary 😅